### PR TITLE
fixed documentation type and TypeError on normalizeEmail() method

### LINKED ISSUE #65: "toDate" example is titled "toInt" in documentation
image

### PR DESCRIPTION
**Pull request by nicklaw5**
Fixes #65 and #67

### CODE REVIEW

File: src/Sanitization/filters.js [hunk 1]
```diff
@@ -109,17 +109,19 @@ SanitizationFilters.normalizeEmail = function (value, args) {
...
-      options.lowercase = false
-    }
-    if (option === '!rd') {
-      options.removeDots = false
-    }
-    if (option === '!re') {
-      options.removeExtension = false
-    }
-  })
+  if(typeof args === 'object') {
```
Comment (thetutlage) [inline src/Sanitization/filters.js]: Can u change the check to be more specific to array. `if(args instanceof Array)`

**Comment by nicklaw5** [comment]
Sorry I'm new to open source contributing, wasn't sure how to undo my last commit. So feel free to squash those last two requests.

Also don't know why the Travis build failed, sorry.

File: src/Sanitization/filters.js [hunk 2]
```diff
@@ -109,17 +109,19 @@ SanitizationFilters.normalizeEmail = function (value, args) {
...
-      options.lowercase = false
-    }
-    if (option === '!rd') {
-      options.removeDots = false
-    }
-    if (option === '!re') {
-      options.removeExtension = false
-    }
-  })
+  if(args instanceof Array)
```
Comment (thetutlage) [inline src/Sanitization/filters.js]: There is no opening curly brace

**Comment by thetutlage** [comment]
No issues, feel free to add your commits. I will squash them

**Comment by nicklaw5** [comment]
added missing bracket, sorry

**Comment by thetutlage** [comment]
@nicklaw5 The PR looks great. Can you also add a test which shows that everything works fine, when `2nd argument` has not been passed. 

The test will go under `test/sanitization.spec.js` file.

**Comment by thetutlage** [comment]
@nicklaw5 Thanks :)

**Pull request merged**